torchgeo/datamodules/geo.py

```diff
@@ -23,15 +23,76 @@
 from .utils import MisconfigurationException
 
 
-class GeoDataModule(LightningDataModule):  # type: ignore[misc]
-    """Base class for data modules containing geospatial information.
+class DataModule(LightningDataModule):  # type: ignore[misc]
```

Comment:
Should we call this a TorchgeoDataModule or something?

Reply:
Could, I don't think it matters too much. The user should never use this class.